assets/styles/global.scss

```diff
@@ -468,9 +468,9 @@ a.has-text-grey {
   cursor: pointer !important;
 }
 
-.has-text-k-accent {
+.has-text-k-primary {
   @include ktheme() {
-    color: theme('k-accent') !important;
+    color: theme('k-primary') !important;
```

Comment:
why did you replace has-text-k-accent
instead of adding the new `has-text-k-primary`?

Reply:
**has-text-k-accent** is only used once , only for the welcome modal icons

Reply:
coolio